x-pack/plugins/actions/server/saved_objects/migrations.ts

```diff
@@ -23,14 +23,15 @@ export function getMigrations(
 ): SavedObjectMigrationMap {
   const migrationActionsTen = encryptedSavedObjects.createMigration<RawAction, RawAction>(
     (doc): doc is SavedObjectUnsanitizedDoc<RawAction> =>
-      !!doc.attributes.config?.casesConfiguration || doc.attributes.actionTypeId === '.email',
+      doc.attributes.config?.hasOwnProperty('casesConfiguration') ||
```

Comment:
Just wanted to call out the reason for these changes. If the value for `doc.attributes.config?.casesConfiguration
` was `null` the expression would previously evaluate to `false` which would skip the migrations. By using `hasOwnProperty` we will check if the field exists at all and perform the migration to remove it.